successstories/index.html

```diff
@@ -0,0 +1,149 @@
+                {{/achievements_arr}}
+            </ul>
+            <hr>
+            <div class="row details-box">
+                <div class="mentee col">
+                    <div class="row mentee-details">
+                        <div>
+                            <div class="mentee-profile-pic">
+                                <img class="img" src="{{mentee_image}}">
+                            </div>
+                        </div>
+                        <div>
+                            <h5 class="name"><a href="{{mentee_linkedin}}">{{mentee_name}}</a></h5>
+                            <p class="details">{{program_year}} Graduate</p>
+                        </div>
+    
+                    </div>
+                </div>
+                <div class="mentor col">
+                    <h5 class="mentor-details-title">Mentored by :</h5>
+                    <div class="row mentor-details">
+                        <div>
+                            <div class="mentor-profile-pic">
+                                <img class="img" src="{{mentor_image}}">
+                            </div>
+                        </div>
+                        <div>
+                            <p class="name"><a href="{{mentor_linkedin}}">{{mentor_name}}</a></p>
+                            <p class="details">{{mentor_details_occupation}}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {{/stories}}
+</script>
+
+<!-- Core -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
+<!-- Optional JS -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<!-- Argon JS -->
+<script src="/assets/js/argon.js?v=1.0.1"></script>
+<!-- Script for static navbar and footer -->
+<script src="/assets/js/navigation.js"></script>
+<!-- External JS file for the JS functions-->
+<script src="functions.js"></script>
+
+<script>
+    const data_url = 'https://script.google.com/macros/s/AKfycbzngFXQxBlsnE6GZ9YeSi5hzFUFuTEBeyy1EgK5w0JN2ixJrtdfPNC6yVOV3gP1VQnGXA/exec';
+
+    async function getStories() {
+        const response = await fetch(data_url);
+        const {data} = await response.json();
+        data.forEach(element => {
+            const{ mentee_name, mentee_linkedin, program_year, mentor_name, mentor_details_occupation, mentor_linkedin, highlight, story, achievements, mentee_image, mentor_image } = element;
+
+            let achievements_arr = achievements.split("\\n");
```

Comment:
```suggestion
            let achievements = achievements.split("\\n");
```

Reply:
the variable called "achievements" is already declared in line 136. Thats why I've used a different name.

Reply:
then reassign it?

Reply:
yeah. That can be done.